modules/ducktests/tests/ignitetest/tests/discovery_test.py

```diff
@@ -169,6 +177,9 @@ def _perform_node_fail_scenario(self, test_config):
             if LATEST_2_7 < test_config.version <= V_2_9_0:
                 discovery_spi.so_linger = 0
 
+            if test_config.disable_conn_recovery:
+                discovery_spi.connRecoveryTimeout = 0
```

Comment:
any guarantee it's not null before? :)
should we use explicit connRecoveryTimeout instead?

Reply:
null is checked in the template. pulint doesn't like many params. Gotta be disabled every time. Think it is ok for simple config.